internal/pkg/releasenotes/release_notes.go

```diff
@@ -67,30 +68,32 @@ func NewFromBody(body string) *ReleaseNotes {
 
 	currentSection := ""
 	for _, line := range strings.Split(body, "\n") {
+		line = strings.TrimSpace(line)
+
 		if utils.Contains(sections, line) {
 			currentSection = line
+			continue
 		}
 
-		if strings.HasPrefix(line, "- ") {
-			note := strings.TrimPrefix(line, "- ")
-
-			if note == "PLACEHOLDER" {
-				continue
-			}
-
-			switch currentSection {
-			case majorSectionTitle:
-				r.major = append(r.major, note)
-			case minorSectionTitle:
-				r.minor = append(r.minor, note)
-			case patchSectionTitle:
-				r.patch = append(r.patch, note)
+		if currentSection != "" {
+			if strings.HasPrefix(line, "- ") {
+				note := strings.TrimPrefix(line, "- ")
+				if note == "PLACEHOLDER" {
```

Comment:
the `<>` around placeholder will be ignored?

Reply:
I changed the template to remove the `<>` because it wasn't showing up on GitHub